### PR TITLE
Udate CHANGES

### PR DESCRIPTION
**Pull request by samoht**
/cc @talex5 and @avsm

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+**all**
+- rename docker/datakit into moby/datakit (#528, @talex5)
+- use jbuilder to build the project (#532, @samoht)
+
+**datakit-ci**
+
+- ci: record history of builds (#489, @talex5)
+- ci: show queue lengths for resource pools in web UI (#500, @talex5)
+- ci: allow separating stdout and stderr in CI_process (#502, @talex5)
+- ci: remove the iframe with the logs (#501, @samoht)
```
Comment (avsm) [inline CHANGES.md]: RIP iframe, you will be missed

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,44 @@
+### 0.10.0 (2017-04-28)
+
+DataKit has a new home: It now lives in the Moby Project!
+
+**all**
+- rename docker/datakit into moby/datakit (#528, @talex5)
+- use jbuilder to build the project (#532, @samoht)
```
Comment (avsm) [inline CHANGES.md]: brief note here on the build speedup and what were the linking bugs you mentioned were fixed @samoht?

**Comment by talex5** [comment]
Maybe note that the images now live in https://hub.docker.com/u/datakit/ ?

**Comment by samoht** [comment]
Update with feedback.

**Pull request merged**